### PR TITLE
Reddeer tests fixing

### PR DESCRIPTION
**Pull request by olkornii**
# Pull Request Checklist

## Code
* Are method-/class-/variable-names meaningful?
* Are methods concise, not too long?
* Are catch blocks catching precise Exceptions only (no catch all)?
* Is copyright with correct year in new files?
* Is there a JavaDoc in every class with description of a class and an author(and issue if suitable)?
* Is there a link to github project issue? (if needed)

## Testing
* Does it work?
* Are the tests green on Jenkins? Is there a link to Jenkins build?

### CODE REVIEW

**Comment by olkornii** [comment]
testPR

**Comment by olkornii** [comment]
testPR

**Comment by olkornii** [comment]
testPR

**Comment by olkornii** [comment]
testPR

**Comment by olkornii** [comment]
testPR

**Comment by olkornii** [comment]
testPR

**Comment by olkornii** [comment]
testPR

**Comment by olkornii** [comment]
testPR

**Comment by olkornii** [comment]
testPR

**Comment by olkornii** [comment]
testPR

**Comment by olkornii** [comment]
testPR

**Comment by olkornii** [comment]
testPR

**Comment by olkornii** [comment]
testPR

**Comment by olkornii** [comment]
testPR

**Comment by olkornii** [comment]
testPR

File: plugins/org.eclipse.reddeer.eclipse/src/org/eclipse/reddeer/eclipse/m2e/core/ui/preferences/MavenPreferencePage.java [hunk 1]
```diff
@@ -212,4 +248,63 @@ public MavenPreferencePage setHideFoldersOfPhysicalyNestedModules(boolean check)
...
+	public boolean isUseMavenCentralToIdentifyUnkArtifChecked() {
+		return new CheckBox(this, MavenPreferencePage.USE_MAVEN_CENTRAL_TO_IDENTIFY_UNKNOWN_ARTIFACTS).isChecked();
+	}
+	
+	/**
+	 * Sets Use Maven-Central to identify unknown artifacts checkbox .
+	 *
+	 * @param check the new Use Maven-Central to identify unknown artifacts
+	 */
+	public MavenPreferencePage setUseMavenCentralToIdentifyUnkArtif(boolean check) {
```
Comment (odockal) [inline plugins/org.eclipse.reddeer.eclipse/src/org/eclipse/reddeer/eclipse/m2e/core/ui/preferences/MavenPreferencePage.java:266]: Too long naming. I would make it shorten and use javadoc for wider description.

File: plugins/org.eclipse.reddeer.eclipse/src/org/eclipse/reddeer/eclipse/m2e/core/ui/preferences/MavenPreferencePage.java [hunk 2]
```diff
@@ -194,6 +211,25 @@ public MavenPreferencePage setUpdateMavenProjectsOnStartup(boolean check) {
 		return this;
 	}
 	
+	/**
+	 * Returns true when Automatically update Maven projects configuration checkbox is checked .
+	 *
+	 * @return true, if is Automatically update Maven projects configuration checked
+	 */
+	public boolean isAutoUpdateMavenProjectConfigurationChecked() {
```
Comment (odockal) [inline plugins/org.eclipse.reddeer.eclipse/src/org/eclipse/reddeer/eclipse/m2e/core/ui/preferences/MavenPreferencePage.java:219]: same as the rest.

File: plugins/org.eclipse.reddeer.eclipse/src/org/eclipse/reddeer/eclipse/m2e/core/ui/preferences/MavenPreferencePage.java [hunk 3]
```diff
@@ -212,4 +248,63 @@ public MavenPreferencePage setHideFoldersOfPhysicalyNestedModules(boolean check)
 		new CheckBox(this, MavenPreferencePage.HIDE_FOLDERS_OF_PHYSICALLY_NESTED_MODULES).toggle(check);
 		return this;
 	}
+	
+	/**
+	 * Returns true when Use Maven-Central to identify unknown artifacts checkbox is checked .
+	 *
+	 * @return true, if is Use Maven-Central to identify unknown artifacts checked
+	 */
+	public boolean isUseMavenCentralToIdentifyUnkArtifChecked() {
```
[the remark this answers is not in the document]
Comment (olkornii) [reply plugins/org.eclipse.reddeer.eclipse/src/org/eclipse/reddeer/eclipse/m2e/core/ui/preferences/MavenPreferencePage.java]: outdated

File: plugins/org.eclipse.reddeer.eclipse/src/org/eclipse/reddeer/eclipse/m2e/core/ui/preferences/MavenPreferencePage.java (hunk 2 above)
Comment (olkornii) [reply plugins/org.eclipse.reddeer.eclipse/src/org/eclipse/reddeer/eclipse/m2e/core/ui/preferences/MavenPreferencePage.java]: outdated

File: plugins/org.eclipse.reddeer.eclipse/src/org/eclipse/reddeer/eclipse/m2e/core/ui/preferences/MavenPreferencePage.java (hunk 1 above)
Comment (olkornii) [reply plugins/org.eclipse.reddeer.eclipse/src/org/eclipse/reddeer/eclipse/m2e/core/ui/preferences/MavenPreferencePage.java]: outdated

**Comment by olkornii** [comment]
testPR

**Review by odockal — APPROVED** [review]

**Comment by olkornii** [comment]
testPR

**Comment by olkornii** [comment]
testPR

**Comment by olkornii** [comment]
testPR

**Pull request merged**